src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
```

Comment:
Consider using a variable for the `margin-right` value to maintain consistency and make it easier to change in the future if needed. This is in line with the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) on using variables for repeated values.